screens/MyPlantScreen.js

```diff
@@ -377,41 +454,55 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'center',
     width: '100%',
-    backgroundColor: Colors.darkGreen,
+    backgroundColor: Colors.defaultWhite,
     paddingVertical: 10,
     borderTopLeftRadius: 15,
     borderTopRightRadius: 15,
   },
 
   smallInfoBody: {
     textAlign: 'center',
-    color: Colors.textGrey,
-    margin: 5,
+    color: Colors.tintColor,
+    paddingVertical: 15,
+    paddingHorizontal: 5,
+    // alignItems: 'center',
+  },
+
+  infoContainer: {
+    marginVertical: 25,
+  },
+
+  infoWrapper: {
+    display: 'flex',
+    alignItems: 'flex-start',
   },
 
   infoHeaderWrapper: {
     display: 'flex',
     flexDirection: 'row',
     alignItems: 'center',
-    justifyContent: 'center',
-    width: '100%',
-    backgroundColor: Colors.darkGreen,
-    paddingVertical: 10,
-    borderRadius: 5,
+    // justifyContent: 'center',
+    // width: '100%',
+    // paddingHorizontal: 10,
+    // borderRadius: 5,
+    borderBottomColor: Colors.tintColor,
+    borderBottomWidth: 1,
+    paddingVertical: 3,
     flexShrink: 1,
   },
 
   infoHeader: {
     fontSize: 20,
-    color: Colors.defaultWhite,
+    color: Colors.tintColor,
+    paddingHorizontal: 10,
+    fontWeight: '500',
+    fontFamily: '',
```

Comment:
New design looks really good, thanks for the effort. I do get an unrecognized font-family error though. That is because of this line, if you remove that its already solved. I think it is also better to remove all the commented out lines and empty strings. so that the code looks cleaner. Then we can merge it.

Reply:
oh yeah I will do that quickly. I am just leaving the page like this for now we can decide on better layout when the time comes and better colours.

Reply:
this empty font family was a solution for some phones that was cutting off the last letter of the text . but looks like ios doesn't like that. i will have to find some other solution for that.

Reply:
I am just leaving this few lines from 21 - 35 and a few others because I want to add this animation but theres some error at the moment

Reply:
I dont have font family there?

Reply:
Line 499 is the line you should comment out. But Ali san said that it has some other purpose so maybe its better to wait for him to find solution.